stdlib/asyncio/events.pyi

```diff
@@ -91,7 +91,6 @@ class TimerHandle(Handle):
     def __le__(self, other: TimerHandle) -> bool: ...
     def __gt__(self, other: TimerHandle) -> bool: ...
     def __ge__(self, other: TimerHandle) -> bool: ...
-    def __eq__(self, other: object) -> bool: ...
```

Comment:
After revisiting some implementation details, this is incorrect and needs to be reverted. Mypy treats `==` with `strict_equliaty` differently with `__eq__` defined in class and in `object` super-class.

So, I will revert all `__eq__` changes.